sentry.sublime-project

```diff
@@ -0,0 +1,44 @@
+{
+    "folders":
+    [
+        {
+            "path": ".",
+        }
+    ],
+    "settings":
+    {
+        "tab_size": 4,
```

Comment:
JavaScript code uses 2 spaces. Looks like to get language-specific tab indents, you need [this package](https://github.com/reywood/sublime-project-specific-syntax), which I don't think we can automate installing.

Is this going to be problematic?